### DISCUSSION TITLE
如何配置access-token

### DISCUSSION THREAD

**Discussion by wlwxj** [Mrs4s/go-cqhttp#1466] [Q&A] [answered]
在yml配置文件启用access-token之后,要将access-token配置在哪里,是请求头还是其他地方
求大佬解答,十分感谢

**Comment by daseinem** [comment, marked as answer]
可参考: https://github.com/botuniverse/onebot-11/blob/master/communication/authorization.md

**Reply by wlwxj** [reply]
好的好的，非常感谢

**Comment by YumeMichi** [comment]
请求头里 上面那个

**Reply by wlwxj** [reply]
ok
